build.rs

```diff
@@ -5,9 +5,8 @@ use std::path::Path;
 
 // Must be public so the build script of `std` can call it.
 pub fn main() {
-    match env::var("CARGO_CFG_TARGET_OS").unwrap_or_default().as_str() {
-        "android" => build_android(),
-        _ => {}
+    if env::var("CARGO_CFG_TARGET_OS").unwrap_or_default().as_str() == "android" {
+        build_android()
     }
```

Comment:
Please do not touch this code, it is slated for deletion.

Reply:
@workingjubilee the whole crate?  Or just the `build.rs`?

Reply:
I reverted this specific instance, hoping it is only `build.rs` that's affected.

Reply:
the build.rs